src/redux-middleware/pullQueue.ts

```diff
@@ -50,14 +50,17 @@ const appendVisibleContexts = (state: State, pullQueue: Index<Context>, visibleC
 
   return keyValueBy(expandedKeys, key => {
     const context = visibleContexts[key]
-    const children = getAllChildrenByContextHash(state, key)
+    const showContexts = isContextViewActive(state, context)
```

Comment:
> I believe this can go outside of the `keyValueBy` loop.

I don't think this should go outside the loop. We need to determine if the each context in the `expandedKeys` have active context view.